testng/src/main/java/cucumber/api/testng/TestNGCucumberRunner.java

```diff
@@ -58,7 +59,7 @@ public void runCukes() {
         }
     }
 
-    public void runCucumber(CucumberFeature cucumberFeature) {
+    public void runFeature(CucumberFeature cucumberFeature) {
```

Comment:
Do not change this method name, it makes the PR contain breaking changes for those people that use the structure of the RunCukesByFeatureAndCompositionTest example runner in the java-calculator-testng example.